__tests__/Utilities.test.js

```diff
@@ -1,6 +1,8 @@
 // Copyright 2019 Stanford University see Apache2.txt for license
 
-import {isResourceWithValueTemplateRef, resourceToName } from '../src/Utilities'
+import 'jsdom-global/register'
+import {isResourceWithValueTemplateRef, resourceToName, generateTitle } from '../src/Utilities'
+import renderer from 'react-test-renderer'
```

Comment:
I don't think we're necessarily doing this with any consistency, but I have noticed a general trend of placing third-party `import`s above local ones. So I'd expect to see our own utils import as the bottom one in this list. This is admittedly very minor!

Reply:
(Also applies to other source and test files touched below.)

Reply:
About jsdom-global: [The docs](https://github.com/rstacruz/jsdom-global#es2015) say to "Place it on top of your other import calls." So my own pattern has been doing that, then our local ones, then other 3rd party ones, but I'm happy to standardize it however folks agree, and for now (in the face of no dissent) I'll change it to how you suggest.